lib/services/runtime/handlers/interaction.ts

```diff
@@ -44,8 +44,12 @@ export const InteractionHandler: HandlerFactory<Node.Interaction.Node, typeof ut
     // check if there is a choice in the node that fulfills intent
     node.interactions.forEach((choice, i: number) => {
```

Comment:
```suggestion
    node.interactions.forEach((choice, i) => {
```
this should be inferred from the builtin `.forEach` type